profiles/SMALL_/profile.mk

```diff
@@ -1,27 +1,38 @@
 
 WORLD_SUBDIRS_LIB+= \
 libalias \
+lib80211 \
```

Comment:
lib80211 should be part of wireless profile. Not all routers have wireless.

Reply:
Current source ifconfig command depend lib80211...

Reply:
Ahh, indeed. sorry for noise

Reply:
This is my check lib dependency script by ruby.  Execute on ZRouter obj directory.

https://gist.github.com/yamori813/b6bc36f6e7373cc231ac0eb5038d1496